lib/mdns_lite/configuration.ex

```diff
@@ -37,6 +37,15 @@ defmodule MdnsLite.Configuration do
     GenServer.start_link(__MODULE__, [], name: __MODULE__)
   end
 
+  @spec add_mdns_services(map() | [map()]) :: :ok
+  def add_mdns_services(service) when is_map(service) do
+    add_mdns_services([service])
+  end
+
+  def add_mdns_services(services) do
```

Comment:
It feels like if there's a way to dynamically add a service that there should be a way to remove one. Or that dynamically added services should be linked to the process that provides them so that when that process dies, the service stops being advertised.

Reply:
Great point. I’ll add a way to remove. That’s easy

Reply:
Added `remove_mdns_services`